### ISSUE TITLE
ADA Compliance on Anchor Links

### ISSUE THREAD

**Issue by andrew-frueh** [HealthCatalyst/Fabric.Cashmere#109]
@corykon and I were discussing this morning, and realized that our $primary-brand blue color for our anchor links is problematic from an accessibility standpoint.  ADA requires a contrast ratio of at least 4.5:1 for normal text.  Even on a white background we're failing that pretty badly:

<img width="737" alt="primarybrandblue" src="https://user-images.githubusercontent.com/22795893/36040357-6fee74ea-0d82-11e8-8eb1-74bd119e0b78.png">

We could switch our anchors to $dark-blue, which does past the contrast test.  But as Cory pointed out, that color doesn't stand out nearly as well within a block of text:

![inparagraph](https://user-images.githubusercontent.com/22795893/36040468-bd98f3c8-0d82-11e8-8062-913c7a88add3.png)

There isn't an obvious solution to this problem to me, but I wanted to include it as an open issue and see if anyone else has ideas.  @cedar-ave - is this something you've addressed at all in your documentation work?
